Application/Frontend/components/FriendsColumn/FooterProfile.tsx

```diff
@@ -18,32 +18,30 @@ import { useRouter } from 'next/router';
  * and settings, encapsulated within a card layout for distinct visual separation.
  */
 export function FooterProfile() {
-
+  const { user } = useUser();
   const router = useRouter();
 
   const handleUserSettingsClick = () => {
-    router.push('//log-in');
+    router.push('/');
   };
 
   return (
-    <>
-      <Card>
-        <Group justify="space-between">
-          <Group>
-            <AppLink href="/">
-              <Tooltip label="Profile">
-                <Avatar radius="xl"/>
-              </Tooltip>
-            </AppLink>
-            <Text>User 1</Text>
-          </Group>
-          <Tooltip label="User Settings">
-            <ActionIcon variant="default" aria-label="Plus" onClick = {handleUserSettingsClick}>
-              <IconSettings style={{ width: '70%', height: '70%' }} stroke={1.5} />
-            </ActionIcon>
-          </Tooltip>
+    <Card>
+      <Group justify="space-between">
+        <Group>
+          <AppLink href="/">
+            <Tooltip label="Profile">
+              <Avatar radius="xl"/>
+            </Tooltip>
+          </AppLink>
+          <Text>{user?.username}</Text>
```

Comment:
The question mark is actually just to satisfy TypeScript's strict type checking because in reality it is guaranteed that user is never null because it is impossible to access the application if you are not signed in. In addition, the if statement within the `useEffect` hook is there to accommodate the short time delay (< 0.5s) between the time component mounts and the time the provider is able to make the data available to us.